osbs/build/build_request.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, build_json_store):
         self._resource_limits = None
         self._openshift_required_version = parse_version('1.0.6')
         # For the koji "scratch" build type
-        self.scratch = False
+        self.scratch = None
```

Comment:
This is done to make sure config param doesn't get overwritten by default params.